server/node/src/main/java/org/kaaproject/kaa/server/operations/service/delta/DefaultDeltaService.java

```diff
@@ -351,10 +363,12 @@ public BaseData compute(List<EndpointGroupStateDto> key) {
         return mergedConfiguration;
     }
 
-    private ConfigurationCacheEntry buildBaseResyncDelta(String endpointId, BaseData mergedConfiguration, EndpointObjectHash userConfigurationHash) {
-        byte[] configuration = GenericAvroConverter.toRawData(mergedConfiguration.getRawData(), mergedConfiguration.getSchema()
-                .getRawSchema());
+    private ConfigurationCacheEntry buildBaseResyncDelta(String endpointId, AbstractKaaData<?> mergedConf, EndpointObjectHash userConfigurationHash) throws IOException {
+        JsonNode json = new ObjectMapper().readTree(mergedConf.getRawData());
+        AvroUtils.removeUuids(json);
```

Comment:
Separate processing of raw and base configuration records according to the 'useConfigurationRawSchema' parameter.